.eslintrc.js

```diff
@@ -191,5 +162,42 @@ module.exports = {
         'react/prop-types': 'off',
       },
     },
+    {
+      files: ['*.tsx'],
+      rules: {
+        'react/prop-types': 'off',
+      },
+    },
+    {
+      files: ['*.spec.tsx', '*.spec.ts'],
+      rules: {
+        '@typescript-eslint/no-unused-expressions': 'off',
+        '@typescript-eslint/no-unused-vars': 'off',
+        '@typescript-eslint/no-use-before-define': 'off',
+        'import/prefer-default-export': 'off',
+        'jsx-a11y/anchor-has-content': 'off',
```

Comment:
This means we treat source files in TS different than JS source files with regards to a11y. I don't think this was the intention. JS and TS should have the same rules for code quality. The only difference should be rules that have counterparts for JS files like [`no-useless-constructor`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-useless-constructor.md)

Reply:
Mind the header: `files: ['*.spec.tsx', '*.spec.ts'],`

Reply:
In a later comment, you said that we should disable `jsx-a11y` for TypeScript tests as we do for end-to-end tests. I'm making the concern resolved.

Reply:
Looked like it was part of `*.tsx`.